sdks/python/apache_beam/runners/dataflow/dataflow_job_service_test.py

```diff
@@ -48,6 +48,7 @@ def test_dry_run(self):
           temp_location='gs://bucket/dir',
           region='us-central1',
           dry_run=True,
+          template_location='gs://bucket/dir',
```

Comment:
Can we split this into a separate test? In theory dry_run should work _and_ a non-dry run with template_location should work.

Reply:
I can do that, but what should i put for the bucket location?